### ISSUE TITLE
Allative

### ISSUE THREAD

**Issue by hectoralos** [apertium/apertium-chv#3]
A suffix АллА is called "allative". The problem is that the vowels previous to it are not deleted or no epenthetic consonants are added:
* `пулӑ<n><all>` = _пуллӑалла_ instead of _пуллалла_
* `пулӑ<n><pl><all>` = _пулӑсеелле_ instead of _пулӑсенелле_
* `лаша<n><all>` = _лашаалла_ instead of _лашалла_
* `лаша<n><pl><all>` = _лашасеелле_ instead of _лашасенелле_
* `музей<n><all>` = _музейелле_ or _музейалла_ instead of _музеелле_ or _музеялла_

It could be added that, according to Chuvash grammarians, this is the dative case + (л)лА. This can help understand the rules about duplication of л in _пул**л**алла_ and the н in _пулӑсе**н**елле_.

**Comment by ftyers** [comment]
Fixed in:

```
 ^пуллалла/пулӑ<n><all>$
 ^пулӑсенелле/пулӑ<n><pl><all>$
 ^лашалла/лашалла<adv>$
 ^лашасенелле/лаша<n><pl><all>$
 ^музеелле/музей<n><all>/музей<n><px2sg><all>$ 
```

Thanks for the 'dative' + (л)лА tip! 

Mostly fixed in a47e918 .

**Comment by hectoralos** [comment]
Gràcies!
